<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Holger Mairon</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1073</width>
    <height>869</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>-  DKV2 -</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QLabel" name="statusLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="EmptyPage">
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background: white</string>
       </property>
       <layout class="QVBoxLayout" name="vertLayout_emptyPage">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>1</number>
        </property>
        <property name="topMargin">
         <number>1</number>
        </property>
        <property name="rightMargin">
         <number>1</number>
        </property>
        <property name="bottomMargin">
         <number>1</number>
        </property>
        <item>
         <widget class="QLabel" name="lblInfo">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Verdana</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">background: white</string>
          </property>
          <property name="text">
           <string>&lt;h1&gt;&lt;br&gt;Willkommen bei  DKV2&lt;/h1&gt;</string>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblSplash">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="resource.qrc">:/res/splash.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="CreditorsListPage">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0">
          <property name="spacing">
           <number>7</number>
          </property>
          <item>
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Filter</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="le_CreditorsFilter">
            <property name="toolTip">
             <string>Gib 2 oder mehr Zeichen ein und drücke die Rücklauftaste</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_reset_filter_creditors">
            <property name="text">
             <string>alle anzeigen</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pbCreditorsColumnsOnOff">
            <property name="text">
             <string>Spalten ein-/ausblenden</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="CreditorsTableView">
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="toolTip">
           <string>Doppelklick oder F2 um ein Feld zu ändern. Rechte Maustaste für weitere Optionen</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="ContractsListPage">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Filter</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="le_ContractsFilter"/>
          </item>
          <item>
           <widget class="QPushButton" name="reset_contracts_filter">
            <property name="text">
             <string>alle anzeigen</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnSave2Csv">
            <property name="text">
             <string>Als Csv Datei speichern</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnVertragsSpalten">
            <property name="text">
             <string>Spalten ein-/ausblenden</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QTableView" name="contractsTableView">
           <property name="contextMenuPolicy">
            <enum>Qt::CustomContextMenu</enum>
           </property>
           <property name="toolTip">
            <string>Verwende die sekundäre Maustaste für weitere Optionen</string>
           </property>
          </widget>
          <widget class="QTableView" name="bookingsTableView"/>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Overview">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="5,1">
          <item>
           <widget class="QComboBox" name="comboUebersicht">
            <property name="editable">
             <bool>false</bool>
            </property>
            <property name="currentIndex">
             <number>-1</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pbPrint">
            <property name="text">
             <string>&amp;speichern</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextEdit" name="txtOverview">
          <property name="undoRedoEnabled">
           <bool>false</bool>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Statistics">
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QGroupBox" name="gbContractRadioBoxes">
            <property name="title">
             <string/>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QRadioButton" name="rbActive">
               <property name="text">
                <string>Aktive Verträge</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rbInactive">
               <property name="text">
                <string>Inaktive Verträge</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rbFinished">
               <property name="text">
                <string>Beendete Verträge</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rbAll">
               <property name="text">
                <string>Alle Verträge</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_13" stretch="0,1,0,0,6">
          <item>
           <widget class="QPushButton" name="pbBack">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string> &lt; &amp;Frühere Buchungstage </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblBookingDate">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>  2021-04-01  </string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pbNext">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string> &amp;Spätere Buchungstage &gt; </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pbLetzter">
            <property name="text">
             <string>&amp;Jüngste Buchung &gt;&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="tvData"/>
        </item>
        <item>
         <widget class="QGraphicsView" name="graphicsView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="InvestmentsPage">
       <layout class="QVBoxLayout" name="verticalLayout_41">
        <item>
         <layout class="QHBoxLayout" name="horicontalLayout">
          <item>
           <widget class="QPushButton" name="btnNewInvestment">
            <property name="toolTip">
             <string>Mit dieser Funktion wird eine neue Geldanlage angelegt</string>
            </property>
            <property name="text">
             <string>Ne&amp;u </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnCreateFromContracts">
            <property name="toolTip">
             <string>Zu jedem Vertrag, zu dem es keine passende Geldanlage gibt, wird eine angelegt.</string>
            </property>
            <property name="text">
             <string>Aus Verträgen &amp;erstellen</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnAutoMatch">
            <property name="toolTip">
             <string>Verträge, für die es nur eine passende Geldanlage gibt, werden dieser zugeordnet</string>
            </property>
            <property name="text">
             <string>Vertr. autom. &amp;zuordnen</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnAutoClose">
            <property name="text">
             <string>Abgelaufene &amp;schließen</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnAlleLoeschen">
            <property name="text">
             <string>Alle löschen</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="InvestmentsTableView"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1073</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_database">
    <property name="title">
     <string>&amp;Datenbank</string>
    </property>
    <addaction name="action_menu_database_start"/>
    <addaction name="separator"/>
    <addaction name="action_menu_database_new"/>
    <addaction name="separator"/>
    <addaction name="action_menu_database_copy"/>
    <addaction name="action_menu_database_anonymous_copy"/>
    <addaction name="separator"/>
    <addaction name="actionProjektkonfiguration_ndern"/>
    <addaction name="action_menu_database_configure_outdir"/>
    <addaction name="separator"/>
    <addaction name="actionZuletzt_verwendet"/>
    <addaction name="separator"/>
    <addaction name="action_menu_database_program_exit"/>
   </widget>
   <widget class="QMenu" name="manu_creditors">
    <property name="title">
     <string>&amp;Kreditgeber</string>
    </property>
    <addaction name="action_menu_creditors_listview"/>
    <addaction name="actionNeu"/>
   </widget>
   <widget class="QMenu" name="manu_contracts">
    <property name="title">
     <string>&amp;Verträge</string>
    </property>
    <widget class="QMenu" name="menu_contracts_subm_print_lists">
     <property name="title">
      <string>&amp;Listen drucken</string>
     </property>
     <addaction name="action_menu_contracts_print_lists"/>
     <addaction name="action_menu_contracts_interestLetters"/>
    </widget>
    <addaction name="action_menu_contracts_listview"/>
    <addaction name="action_menu_Beendete_Vertr_ge_anzeigen"/>
    <addaction name="action_Neu"/>
    <addaction name="separator"/>
    <addaction name="action_menu_contracts_annual_interest_settlement"/>
    <addaction name="separator"/>
    <addaction name="actionAnlagen_verwalten"/>
    <addaction name="action_menu_contracts_statistics_view"/>
    <addaction name="actionStatistik"/>
    <addaction name="menu_contracts_subm_print_lists"/>
   </widget>
   <widget class="QMenu" name="menu_debug">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>&amp;Extras</string>
    </property>
    <addaction name="action_menu_debug_create_sample_data"/>
    <addaction name="action_menu_debug_show_log"/>
    <addaction name="separator"/>
    <addaction name="actionDatenbank_Views_schreiben"/>
    <addaction name="separator"/>
    <addaction name="action_about_DKV2"/>
   </widget>
   <addaction name="menu_database"/>
   <addaction name="manu_creditors"/>
   <addaction name="manu_contracts"/>
   <addaction name="menu_debug"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_menu_database_new">
   <property name="text">
    <string>&amp;Wechseln...</string>
   </property>
   <property name="toolTip">
    <string>Legen Sie hiermit eine leere DK Datenbank an</string>
   </property>
  </action>
  <action name="action_menu_creditors_listview">
   <property name="text">
    <string>&amp;Liste </string>
   </property>
   <property name="toolTip">
    <string>Zeigt eine Liste aller DK Geber an</string>
   </property>
  </action>
  <action name="action_menu_database_program_exit">
   <property name="text">
    <string>Programm beenden</string>
   </property>
  </action>
  <action name="action_menu_contracts_listview">
   <property name="text">
    <string>&amp;Liste anzeigen</string>
   </property>
  </action>
  <action name="action_menu_debug_create_sample_data">
   <property name="text">
    <string>Beispieldaten erstellen</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="action_cmenu_delete_creditor">
   <property name="text">
    <string>Löschen</string>
   </property>
   <property name="toolTip">
    <string>Löscht einen Kreditgeber mit seinen Verträgen </string>
   </property>
  </action>
  <action name="actionshowLog">
   <property name="text">
    <string>Show Log file</string>
   </property>
  </action>
  <action name="action_cmenu_activate_contract">
   <property name="text">
    <string>&amp;Geldeingang verbuchen</string>
   </property>
   <property name="toolTip">
    <string>Zinszahlung des Vertrags aktivieren</string>
   </property>
  </action>
  <action name="action_cmenu_delete_inactive_contract">
   <property name="text">
    <string>Vertrag &amp;löschen</string>
   </property>
  </action>
  <action name="action_menu_debug_show_log">
   <property name="text">
    <string>Log Datei öffnen</string>
   </property>
  </action>
  <action name="action_cmenu_terminate_contract">
   <property name="text">
    <string>Vertrag &amp;beenden</string>
   </property>
   <property name="toolTip">
    <string>Zins und Auszahlungsbetrag berechnen, Vertrag löschen</string>
   </property>
  </action>
  <action name="action_menu_contracts_statistics_view">
   <property name="text">
    <string>Übe&amp;rsichten</string>
   </property>
   <property name="toolTip">
    <string>Übersicht über Vertragsdaten anzeigen</string>
   </property>
  </action>
  <action name="action_cmenu_go_contracts">
   <property name="text">
    <string>Verträge zeigen</string>
   </property>
   <property name="toolTip">
    <string>zu den Verträgen dieses Kreditgebers wechseln</string>
   </property>
  </action>
  <action name="action_cmenu_edit_creditor">
   <property name="text">
    <string>Bearbeiten</string>
   </property>
   <property name="toolTip">
    <string>Adressdaten und Bankdaten bearbeiten</string>
   </property>
  </action>
  <action name="action_menu_contracts_annual_interest_settlement">
   <property name="text">
    <string>&amp;Jahreszinsabrechnung...</string>
   </property>
  </action>
  <action name="action_menu_database_configure_outdir">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Ausgabeverzeichnis festlegen...</string>
   </property>
  </action>
  <action name="action_menu_contracts_print_lists">
   <property name="text">
    <string>Aktive Verträge mit Kontaktdaten</string>
   </property>
  </action>
  <action name="action_menu_database_start">
   <property name="text">
    <string>Start</string>
   </property>
  </action>
  <action name="action_menu_database_anonymous_copy">
   <property name="text">
    <string>Anonymisierte &amp;Kopie...</string>
   </property>
  </action>
  <action name="action_menu_database_copy">
   <property name="text">
    <string>Kopie anlegen...</string>
   </property>
  </action>
  <action name="action_about_DKV2">
   <property name="text">
    <string>Über DKV2</string>
   </property>
  </action>
  <action name="action_cmenu_change_contract">
   <property name="text">
    <string>Ein- oder Auszahlung</string>
   </property>
   <property name="toolTip">
    <string>Ein- oder Auszahlung</string>
   </property>
  </action>
  <action name="actionDatenbank_Views_schreiben">
   <property name="text">
    <string>Datenbank Views schreiben</string>
   </property>
  </action>
  <action name="action_menu_Beendete_Vertr_ge_anzeigen">
   <property name="text">
    <string>&amp;Beendete V. anzeigen</string>
   </property>
  </action>
  <action name="actionProjektkonfiguration_ndern">
   <property name="text">
    <string>Projektkonfiguration ändern...</string>
   </property>
  </action>
  <action name="actionNeu">
   <property name="text">
    <string>&amp;Neu anlegen...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_Neu">
   <property name="text">
    <string>&amp;Neu anlegen...</string>
   </property>
  </action>
  <action name="actionAktuelle_Auswahl">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Vertragstabelle (gefiltert)</string>
   </property>
  </action>
  <action name="actionTEST">
   <property name="text">
    <string>TEST</string>
   </property>
  </action>
  <action name="actionAnlagen_verwalten">
   <property name="text">
    <string>&amp;Anlagen verwalten</string>
   </property>
  </action>
  <action name="actionInvestmentLoeschen">
   <property name="text">
    <string>Löschen</string>
   </property>
   <property name="toolTip">
    <string>Das Investment wird aus der Datenbank gelsöcht. </string>
   </property>
  </action>
  <action name="actionInvestmentSchliessen">
   <property name="text">
    <string>Investment status umschalten</string>
   </property>
   <property name="toolTip">
    <string>Zu einer geschlossenen Geldanlage kann kein Vertrag mehr angelegt werden.</string>
   </property>
  </action>
  <action name="actionTyp_Bezeichnung_aendern">
   <property name="text">
    <string>&amp;Bezeichnung der Anlage ändern</string>
   </property>
   <property name="toolTip">
    <string>Mit dieser Option kannst Du den Typ ändern</string>
   </property>
  </action>
  <action name="actionStatistik">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>V&amp;erlauf</string>
   </property>
  </action>
  <action name="action_cmenu_Anmerkung_aendern">
   <property name="text">
    <string>Anmerkung ändern</string>
   </property>
  </action>
  <action name="action_cmenu_changeContractTermination">
   <property name="text">
    <string>&amp;Kündigungsfrist und Vertragsende bearbeiten</string>
   </property>
  </action>
  <action name="action_cmenu_assoc_investment">
   <property name="text">
    <string>Geldanlage zuordnen</string>
   </property>
  </action>
  <action name="action_cmenu_activate_interest_payment">
   <property name="text">
    <string>Zinszahlung aktivieren</string>
   </property>
  </action>
  <action name="action_cmenu_Vertraege_anzeigen">
   <property name="text">
    <string>Verträge anzeigen</string>
   </property>
   <property name="toolTip">
    <string>Zeigt alle Verträge, die zu dieser Anlage abgeschlossen wurden</string>
   </property>
  </action>
  <action name="action_menu_contracts_interestLetters">
   <property name="text">
    <string>Zinsbriefe</string>
   </property>
   <property name="toolTip">
    <string>Zinsbriefe als  PDF ausgeben</string>
   </property>
  </action>
  <action name="actionZuletzt_verwendet">
   <property name="text">
    <string>Zuletzt verwendet</string>
   </property>
  </action>
  <action name="action_cmenu_contracts_EndLetter">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Endabrechnung</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Brief und CSV zur Endabrechnung ausgeben&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="action_cmenu_change_label">
   <property name="text">
    <string>Vertragskennung ändern</string>
   </property>
  </action>
  <action name="action_cmenu_change_conclusion_date">
   <property name="text">
    <string>Vertragsdatum ändern</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>le_CreditorsFilter</tabstop>
  <tabstop>comboUebersicht</tabstop>
  <tabstop>txtOverview</tabstop>
  <tabstop>CreditorsTableView</tabstop>
  <tabstop>btn_reset_filter_creditors</tabstop>
  <tabstop>le_ContractsFilter</tabstop>
  <tabstop>reset_contracts_filter</tabstop>
  <tabstop>btnSave2Csv</tabstop>
  <tabstop>btnVertragsSpalten</tabstop>
  <tabstop>contractsTableView</tabstop>
  <tabstop>bookingsTableView</tabstop>
  <tabstop>rbActive</tabstop>
  <tabstop>rbInactive</tabstop>
  <tabstop>rbFinished</tabstop>
  <tabstop>rbAll</tabstop>
  <tabstop>pbBack</tabstop>
  <tabstop>pbNext</tabstop>
  <tabstop>pbLetzter</tabstop>
  <tabstop>tvData</tabstop>
  <tabstop>graphicsView</tabstop>
  <tabstop>btnNewInvestment</tabstop>
  <tabstop>btnCreateFromContracts</tabstop>
  <tabstop>btnAutoMatch</tabstop>
  <tabstop>btnAutoClose</tabstop>
  <tabstop>InvestmentsTableView</tabstop>
  <tabstop>pbPrint</tabstop>
 </tabstops>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
